client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -366,7 +368,7 @@ abstract class BaseActivity(
     /**
      * Hide the keyboard
      */
-    protected fun hideKeyboard() {
+    fun hideKeyboard() {
```

Comment:
This can also be removed and can be converted to an extension method instead.

Reply:
Yes